docs/rustdoc/centralized_telescope/main.md

```diff
@@ -1,3 +1,55 @@
 # Telescope - Construction with Bounded DFS
-This is the main page of centralized documentation.
-It will include a generic explanation of the protocol.
+
```

Comment:
Is this file in `docs/rustdoc/centralized_telescope/` on purpose as opposed to `docs/rustdoc/` like for the basic and prehashed docs?

Reply:
Centralized telescope will include several files. This is the main page, parameter setup and other algorithm explanations will have their own pages as opposed to basic and prehashed docs. So, I decided to store them in a folder.

Reply:
Ok, I see.

Reply:
Since this doc is in a different location, maybe add a link to the prehashed version doc when you mention it at the beginning?